layout/index.hbs

```diff
@@ -5,12 +5,23 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
-    <link rel="icon" type="image/svg+xml" href="./images/favicon.svg" />
+    <!-- browsers -->
+    <link rel="icon" type="image/svg+xml" href="/images/half-day-favicon.svg">
```

Comment:
You should make these `./images/whatever` - helps when dev’ing and the root of your dev webserver isn’t dist/

Reply:
Got it, will do.